src/test/java/uk/nhs/prm/e2etests/test/RepositoryE2ETest.java

```diff
@@ -555,4 +586,26 @@ private void setManagingOrganisationToRepo(String nhsNumber) {
             pdsAdaptorService.updateManagingOrganisation(nhsNumber, repoOdsCode, pdsResponse.getRecordETag());
         }
     }
+
+    // ============ HELPER METHODS ============
+    private void addSmallEhrToEhrRepo(String nhsNumber) {
+        final String inboundConversationId = UUID.randomUUID().toString();
+        final String messageId = UUID.randomUUID().toString();
+        final SmallEhrTemplateContext smallEhrTemplateContext = SmallEhrTemplateContext.builder()
+                .inboundConversationId(inboundConversationId.toUpperCase())
+                .messageId(messageId.toUpperCase())
+                .nhsNumber(nhsNumber)
+                .build();
+        final String smallEhrMessage = this.templatingService.getTemplatedString(SMALL_EHR_WITHOUT_LINEBREAKS, smallEhrTemplateContext);
```

Comment:
```suggestion
        final SmallEhrTemplateContext smallEhrTemplateContext = SmallEhrTemplateContext.builder()
                .nhsNumber(nhsNumber)
                .build();
        final String inboundConversationId = smallEhrTemplateContext.getInboundConversationId().toLowerCase();
        final String smallEhrMessage = this.templatingService.getTemplatedString(SMALL_EHR_WITHOUT_LINEBREAKS, smallEhrTemplateContext);
```

I did some trial & error here, and it seems that we could let SmallEhrTemplateContext to generate the conversation Id. Just that the conversationId needs to be lowercased before passed to transferTrackerService.

...Or we could possibly modify TransferTrackerService and let it automatically lowercase any conversationId that it got from method call.